### PR TITLE
Use correct artifact path when creating a new byte channel

### PR DESCRIPTION
**Pull request by eamansour**
Part of https://github.com/galasa-dev/projectmanagement/issues/1407
Linked to https://github.com/galasa-dev/framework/pull/403

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: galasa-extensions-parent/dev.galasa.ras.couchdb/build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ plugins {
 
 description = 'Galasa RAS - CouchDB'
 
-version = '0.25.0'
+version = '0.26.0'
```
Comment (hobbit1983) [inline galasa-extensions-parent/dev.galasa.ras.couchdb/build.gradle:8]: shouldn't this be 28?

File: release.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ framework:
     codecoverage: true
     
   - artifact: dev.galasa.ras.couchdb
-    version: 0.25.0
+    version: 0.26.0
```
Comment (hobbit1983) [inline release.yaml:19]: 28?

**Review by hobbit1983 — CHANGES_REQUESTED** [review]

File: galasa-extensions-parent/dev.galasa.ras.couchdb/build.gradle (hunk 1 above)
Comment (eamansour) [reply galasa-extensions-parent/dev.galasa.ras.couchdb/build.gradle]: Addressed in https://github.com/galasa-dev/extensions/pull/200/commits/0124979801009e10b7c4fc6f1bce4ef8583e247b

File: release.yaml (hunk 2 above)
Comment (eamansour) [reply release.yaml]: Addressed in https://github.com/galasa-dev/extensions/pull/200/commits/0124979801009e10b7c4fc6f1bce4ef8583e247b

**Review by hobbit1983 — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**